[2023-05-29 00:51:45,804] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ML_data_pipeline_demo.process_raw_data manual__2023-05-29T00:45:51.647870+00:00 [queued]>
[2023-05-29 00:51:45,821] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ML_data_pipeline_demo.process_raw_data manual__2023-05-29T00:45:51.647870+00:00 [queued]>
[2023-05-29 00:51:45,822] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-29 00:51:45,823] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-05-29 00:51:45,824] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-29 00:51:45,844] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_raw_data> on 2023-05-29 00:45:51.647870+00:00
[2023-05-29 00:51:45,854] {standard_task_runner.py:52} INFO - Started process 13829 to run task
[2023-05-29 00:51:45,867] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ML_data_pipeline_demo', 'process_raw_data', 'manual__2023-05-29T00:45:51.647870+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp_awvz9n8', '--error-file', '/tmp/tmpk0ug812u']
[2023-05-29 00:51:45,870] {standard_task_runner.py:80} INFO - Job 5: Subtask process_raw_data
[2023-05-29 00:51:45,958] {task_command.py:369} INFO - Running <TaskInstance: ML_data_pipeline_demo.process_raw_data manual__2023-05-29T00:45:51.647870+00:00 [running]> on host e6406d4dfe47
[2023-05-29 00:51:46,075] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ML_data_pipeline_demo
AIRFLOW_CTX_TASK_ID=process_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2023-05-29T00:45:51.647870+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-29T00:45:51.647870+00:00
[2023-05-29 00:54:02,980] {python.py:173} INFO - Done. Returned value was: None
[2023-05-29 00:54:03,003] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ML_data_pipeline_demo, task_id=process_raw_data, execution_date=20230529T004551, start_date=20230529T005145, end_date=20230529T005403
[2023-05-29 00:54:03,060] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-29 00:54:03,118] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
